pandas/tests/groupby/test_groupby.py

```diff
@@ -1994,3 +1994,22 @@ def test_dup_labels_output_shape(groupby_func, idx):
 
     assert result.shape == (1, 2)
     tm.assert_index_equal(result.columns, idx)
+
+
+def test_groupby_crash_on_nunique(axis):
+    # Fix following 30253
+    df = pd.DataFrame(
+        {("A", "B"): [1, 2], ("A", "C"): [1, 3], ("D", "B"): [0, 0]},
+        index=pd.bdate_range("20191212", "20191213"),
```

Comment:
Unless these need to be date values (I don't think so) probably easier to just leave unspecified; otherwise it distracts a bit from what the test is aiming to fix

Reply:
OK, will do